python/pyspark/ml/clustering.py

```diff
@@ -201,7 +202,74 @@ def computeCost(self, dataset):
         """
         return self._call_java("computeCost", dataset)
 
+    @since("2.1.0")
+    def hasSummary(self):
+        """
+        Return true if there exists summary of model.
+        """
+        return self.summary is not None
```

Comment:
@zjffdu Can you make this match the other summary implementations? i.e. `self._call_java("hasSummary")`